code/game/machinery/vending.dm

```diff
@@ -962,21 +963,22 @@
 
 /obj/machinery/vending/wallmed1
 	name = "\improper NanoMed"
-	desc = "A wall-mounted version of the NanoMed."
+	desc = "A wall-mounted version of the NanoMed, stocked with the equipment necessary for stabilisation and much more."
```

Comment:
If you're going to make a vending machine's description unnecessarily long, at LEAST do it to all variants of that vending machine.

Reply:
this is the one that is used ingame.